components/lib/VHeader.vue

```diff
@@ -104,7 +104,7 @@ header {
 }
 
 .header-menu-item:active {
-  opacity: 0.8;
+  transform: scale(98%);
```

Comment:
Prefer unitless values for scale.

```suggestion
  transform: scale(0.98);
```

Reply:
@punaifang added changes